viewer/src/datamodels/cad/sector/CachedRepository.ts

```diff
@@ -204,25 +200,9 @@ export class CachedRepository implements Repository {
   }
 
   private loadDetailedSectorFromNetwork(wantedSector: WantedSector): Observable<ConsumedSector> {
```

Comment:
```
private loadDetailedSectorFromNetwork(wantedSector: WantedSector): Observable<ConsumedSector> {
    const detailedSectorObservable = from(
      /** Needs to be renamed so that it is clear this also takes over network requests etc.
       * Say we rename it to loadDetailedFromNetwork() and we send in a bit more data.
       * {
       *   blobUrl: wantedSector.blobUrl,
       *   headers: this._modelSectorProvider.headers,
       *   i3dFile: wantedSector.metadata.indexFile.fileName,
       *   ctmFiles: wantedSector.metadata.indexFile.peripheralFiles,
       * }
       *
       * That way you don't have to first request the i3d file, then parse to see what ctm files are needed to download.
       */
      this._modelDataParser.parseAndFinalizeDetailed(wantedSector.metadata.indexFile.fileName, {
        blobUrl: wantedSector.blobUrl,
        headers: this._modelSectorProvider.headers
      })
    ).pipe(
      catchError(error => {
        trackError(error, {
          moduleName: 'CachedRepository',
          methodName: 'loadDetailedSectorFromNetwork'
        });
        this._consumedSectorCache.remove(this.wantedSectorCacheKey(wantedSector));
        throw error;
      }),
      map(finalizedDetailed => {
        const sector = this.unflattenSector(finalizedDetailed);
        this._parsedDataSubject.next({
          blobUrl: wantedSector.blobUrl,
          sectorId: wantedSector.metadata.id,
          lod: 'detailed',
          data: sector
        }); // TODO: Remove when migration is gone.
        return { ...wantedSector, data: sector };
      }),
      this._modelDataTransformer.transform(),
      map(group => ({ ...wantedSector, group })),
      shareReplay(1),
      take(1)
    );
    this._consumedSectorCache.forceInsert(this.wantedSectorCacheKey(wantedSector), detailedSectorObservable);
    return detailedSectorObservable;
  }
```

This entire function can be simplified